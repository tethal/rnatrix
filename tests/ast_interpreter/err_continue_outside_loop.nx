fun main() {
    var x = 10;
    if (x > 5) {
        continue;
    }
}