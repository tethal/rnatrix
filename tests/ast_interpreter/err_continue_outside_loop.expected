<string>:4:9: error: continue outside a loop
        continue;
        ^^^^^^^^^
